### DISCUSSION TITLE
dust attenuation

### DISCUSSION THREAD

**Discussion by JosieLeviosa** [mboquien/cigale#45] [Q&A]
Hello,
I'm using CIGALE to fit high-z galaxies and I was wondering if there's a possibility to use a dust attenuation law other than Calzetti. I'm currently using the module "dustatt_modified_starburst" and as far as I understand it uses Calzetti for the continuum and has the option to use LMC or SMC dust curves for the emission lines. Is it possible to use those for the continuum as well? Or am I missing something?
Many thanks!

**Comment by mboquien** [comment]
Hi! This module tries to stick reasonably closely to the starburst curve. If you want a different shape for the stellar continuum, what you probably want to look at is the delta parameter. If you want a curva that is similar to an LMC curve, δ=-0.3 is appropriate. An SMC curve is quite similar to δ=-0.5. Note that at high z it is not uncommon to get δ<-0.5 when there is a fairly low attenuation (see for instance Boquien+22). I hope this helps.

**Reply by JosieLeviosa** [reply]
Right, thanks so much! It wasn't quite clear to me what the delta parameter did, but then I'll stick to that for changing the curve.

**Reply by JosieLeviosa** [reply]
On a related note, is there a way to get A_v directly from the output of CIGALE?

**Reply by mboquien** [reply]
By default it should be in the `attenuation.V_B90` column.
